configuration/wireless/access-point-bridged.md

```diff
@@ -0,0 +1,162 @@
+    * Be ready to add screen and keyboard in case you lose contact with your Raspberry Pi after installation. 
+* Connect your Raspberry Pi to the Ethernet network and boot the Raspbian OS.
+* Ensure the Raspbian OS on your Raspberry Pi is [up-to-date](../../raspbian/updating.md) and reboot if packages were installed in the process.
+* Have a wireless client (laptop, smartphone, ...) ready to test your new access point.
+
+<a name="software-install"></a>
+## Install the access point software
+
+In order to work as a bridged access point, the Raspberry Pi needs to have the `hostapd` access point software package installed:
+
+```
+sudo apt install hostapd
+```
+Enable the wireless access point service and set it to start when your Raspberry Pi boots:
+
+```
+sudo systemctl unmask hostapd
+sudo systemctl enable hostapd
+```
+
+Software installation is complete. We will configure the access point software later on.
+
+<a name="bridging"></a>
+## Setup the network bridge
+
+A bridge network device running on the Raspberry Pi will connect the Ethernet and wireless networks using its built-in interfaces.
+
+### Create a bridge device and populate the bridge
+
+Add a bridge network device named `br0` by creating a file using the following command, with the contents below:
+
+```
+sudo nano /etc/systemd/network/bridge-br0.netdev
+```
+
+File contents:
+```
+[NetDev]
+Name=br0
+Kind=bridge
+```
+
+In order to bridge the Ethernet network with the wireless network, first add the built-in Ethernet interface (`eth0`) as a bridge member by creating the following file:
+
+```
+sudo nano /etc/systemd/network/br0-member-eth0.network
+```
+
+File contents:
+```
+[Match]
+Name=eth0
+
+[Network]
+Bridge=br0
+```
+
+**Note:** The access point software will add the wireless interface `wlan0` to the bridge when the service starts. There is no need to create a file for that interface. This situation is particular to wireless LAN interfaces.
+
+Now enable the `systemd-networkd` service to create and populate the bridge when your Raspberry Pi boots:
+
+```
+sudo systemctl enable systemd-networkd
+```
+
+### Define the bridge device IP configuration
+
+Network interfaces that are members of a bridge device are never assigned an IP address, since they communicate via the bridge. The bridge device itself needs an IP address, so that you can reach your Raspberry Pi on the network.
+
+`dhcpcd`, the DHCP client on the Raspberry Pi, automatically requests an IP address for every active interface. So we need to block the `eth0` and `wlan0` interfaces from being processed, and let `dhcpcd` configure only `br0` via DHCP.
+
+```
+sudo nano /etc/dhcpcd.conf
+```
+
+Add the following line near the beginning of the file (above the first `interface xxx` line, if any):
+```
+denyinterfaces wlan0 eth0
+```
+Interface `br0` will be configured in accordance with the defaults via DHCP; no specific entry is necessary. Save the file to complete the IP configuration of the machine.
+
```

Comment:
Go to the end of the file and add the following:

```
interface br0

```
With this line, interface `br0` will be configured in accordance with the defaults via DHCP. Save the file to complete the IP configuration of the machine.

Reply:
What's above is a proposed fix, as per #1423.
The routed ap howto already asked users to add something at the beginning of file dhcpcd.conf, and something else at the end of that same file. Now this seems necessary for the bridged ap as well.